framework/platform/bpl/uci/scripts/platform_daemon.sh

```diff
@@ -23,7 +23,7 @@ repeater_to_ap()
     /opt/beerocks/apm_utils.sh stop
 
     # Kill DHCP client
-    ps -A | grep [d]hcp | awk '{print $1;}' | xargs kill
+    pgrep [d]hcp | xargs kill
```

Comment:
I've checked that `pgrep` is present on RAX40 board.
     And absent `pkill` because otherwise i'd use it instead of.
I can check `pgrep` presence in built images for another platforms.

Reply:
I've verified existence of `pgrep` on the UGW and RDKB platforms
`pkill` on the other hand exists in RDKB but not in UGW
@tomereli @kantera800

Reply:
In this case (UGW is much harder to change since it is a real product) - lets not use `pkill`.

Reply:
I've checked that `pgrep` is present in nightly build images for `turris-omnia` and `netgear-rax40`.
Also checked that config option `CONFIG_BUSYBOX_DEFAULT_PGREP=y` is present in `.config` OpenWrt file for device `glinet-b1300`.

So using `pgrep` looks reasonable